node/lib/ILogger.ts

```diff
@@ -1,8 +1,15 @@
 export interface ILogger {
-    error(msg: string, logObject?: any): void;
-    warn(msg: string, logObject?: any): void;
-    info(msg: string, logObject?: any): void;
-    verbose(msg: string, logObject?: any): void;
-    debug(msg: string, logObject?: any): void;
+    // Example of loggingMetadata is correlationVector
+    error(msg: string, logObject?: any, logMetadata?: string): void;
+    warn(msg: string, logMetadata?: string, logObject?: any): void;
+/*    warn(msg: string, logObject?: any, logMetadata?: string): void;
+    warn(msg: string, logParams?: Map<any, string>): void;
+    warn(msg: string, logParams?: Array<any>): void;*/
```

Comment:
Why is this commented out? Remove the comment or explain a comment explaining the comment if the commented out code is really needed.